var/spack/repos/builtin/packages/py-bcrypt/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class PyBcrypt(PythonPackage):
+    """Modern password hashing for your software and your servers"""
+
+    homepage = "https://github.com/pyca/bcrypt/"
+    url      = "https://github.com/pyca/bcrypt/archive/3.1.4.tar.gz"
+
+    version('3.1.4', '2db1e1bf4a9e92f78297e1f090d7a30e')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-cffi', type=('build', 'run'))
+    depends_on('py-six', type=('build', 'run'))
+
```

Comment:
Can you add the specific version ranges?
```python
CFFI_DEPENDENCY = "cffi>=1.1"                                                   
SIX_DEPENDENCY = "six>=1.4.1"   
```
Also, the blank line at the end of the file is going to cause flake8 to complain.